requirements.txt

```diff
@@ -17,7 +17,7 @@ numpy==1.24.3
 openai==0.27.8
 openpyxl
 pandas==1.4.1
-pydantic==1.10.7
+pydantic==1.10.8
```

Comment:
why update the version?
Do you omit the qdrant version?

Reply:
because qdrant-client 1.4.0 requires pydantic>=1.10.8,and the version of pydantic 1.x is compatible, so I changed the version of pydantic,i will add qdrant-client version